src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+spring.application.name=ph-ee-connector-airtel
```

Comment:
if you are using application.yml you can remove this file

Reply:
Done.